cishouseholds/edit.py

```diff
@@ -257,8 +257,8 @@ def clean_barcode(df: DataFrame, barcode_column: str, edited_column: str) -> Dat
     edited_column
         signifies if updating was performed on row
     """
-    df = df.withColumn("BARCODE_COPY", F.col(barcode_column))
     df = df.withColumn(barcode_column, F.upper(F.regexp_replace(F.col(barcode_column), " ", "")))
+    df = df.withColumn("BARCODE_COPY", F.col(barcode_column))
```

Comment:
The line above includes other cleaning, as well as the conversion to upper. I think you could drop the regexp_replace part above, as the line below should also match whitespace

Reply:
Could we also use an object instead of a column for `BARCODE_COPY`?